android/src/main/java/io/mosip/tuvali/verifier/GattService.kt

```diff
@@ -47,38 +48,46 @@ class GattService {
       BluetoothGattCharacteristic.PERMISSION_WRITE
     )
 
-    val responseChar = BluetoothGattCharacteristic(
-      RESPONSE_CHAR_UUID,
+    val submitResponseChar = BluetoothGattCharacteristic(
+      SUBMIT_RESPONSE_CHAR_UUID,
       BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE or BluetoothGattCharacteristic.PROPERTY_WRITE,
       BluetoothGattCharacteristic.PERMISSION_WRITE
     )
 
-    val semaphoreChar = BluetoothGattCharacteristic(
-      SEMAPHORE_CHAR_UUID,
-      BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE or BluetoothGattCharacteristic.PROPERTY_WRITE or BluetoothGattCharacteristic.PROPERTY_READ or BluetoothGattCharacteristic.PROPERTY_INDICATE,
+    val transferReportRequestChar = BluetoothGattCharacteristic(
+      TRANSFER_REPORT_REQUEST_UUID,
+      BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE or
```

Comment:
I think we can remove following for this characteristic

BluetoothGattCharacteristic.PROPERTY_WRITE, BluetoothGattCharacteristic.PROPERTY_READ

&

BluetoothGattCharacteristic.PERMISSION_READ

Reply:
A new github issue is created for this https://github.com/mosip/inji/issues/485

Use this issue number for the commits related to property cleanup @PuBHARGAVI @adityankannan-tw 

/cc @tilak-puli